fixed fontcolor set in class definition not used for SequenceTrack su…

FelixErnst opened the pull request:
…bclasses

The fontcolor setting of the individual subclasses is not used, since it is overwritten by the `.schemes` value.

```
>st <- new("SequenceDNAStringSetTrack")
>displayPars(st)$fontcolor
NULL
```

Now this returns the values set in the class definition.

ivanek commented:
Hi @FelixErnst, 
I am sorry I cannot accept this pull request. The default settings leads to SequenceTrack plots with no colors in the current vignette.
Robert

FelixErnst commented:
Hi Robert @ivanek 

should be fixed now. I deleted all the default settings for `SequenceTrack` classes, which were never applied, since they were overwritten by the `.schemes` value anyway and added those, where they were missing. Settings and the way they are set should now be consistent for all `SequenceTrack` classes.

Felix

ivanek commented:
Hi Felix,

I just accepted your pull request. Thanks for it.

Best
Robert
